src/main/java/dev/technici4n/moderndynamics/network/fluid/FluidCache.java

```diff
@@ -137,36 +128,34 @@ public void doTick() {
 
             if (!fluidStorage.isResourceBlank()) {
                 // Take from connected storages
-                extractFluid(targets, tx);
-                attractFluid(targets, attractors, tx);
+                extractFluid(targets);
+                attractFluid(targets, attractors);
                 // Push to connected storages
-                distributeFluid(targets, tx);
+                distributeFluid(targets);
 
                 if (fluidStorage.amount == 0 && canChangeVariant()) {
                     fluidStorage.variant = FluidVariant.blank();
                     changedVariant = true;
                 }
             }
-
-            tx.commit();
         } finally {
             allowNetworkIo = true;
         }
 
-        // Always separate after a change of variant to ensure that the nodes properly update their stored fluid.
+        // Always separate after a change of variant to ensure that the nodes properly update their stored item.
```

Comment:
Did you actually replace every instance of `fluid` by `item`? Probably an intellij issue. :P